src/CourseDashboardLogIn.cpp

```diff
@@ -0,0 +1,55 @@
+#include "CourseDashboardLogIn.hpp"
+#include <stdexcept>
+#include <iostream>
+#include <algorithm>
+#include <fstream>
+
+CourseDashboardLogIn::CourseDashboardLogIn()
+{}
+
+CourseDashboardLogIn::~CourseDashboardLogIn()
+{}
```

Comment:
Empty constructor and destructor will be generated anyway, they should not be implemented.